symbolic/src/lib.rs

```diff
@@ -73,6 +73,9 @@ pub use symbolic_debuginfo as debuginfo;
 #[doc(inline)]
 #[cfg(feature = "demangle")]
 pub use symbolic_demangle as demangle;
+#[doc(inline, hidden)]
```

Comment:
@Swatinem I've marked this as hidden as well as documented it in the module as do not use.  I think this should be sufficient.